cdap-app-fabric/src/main/java/co/cask/cdap/test/RuntimeStats.java

```diff
@@ -50,34 +58,31 @@ public static void resetAll() throws Exception {
     metricStore.deleteBefore(System.currentTimeMillis() / 1000);
   }
 
+  public static RuntimeMetrics getMapReduceMetrics(String applicationId, String mapReduceId) {
```

Comment:
does wait for works for mapreduce? does any unit-test or any test uses this method? Do you know if it works?